etc/kayobe/kolla/globals.yml

```diff
@@ -15,6 +15,7 @@ nova_tag: wallaby-20220328T103755
 octavia_tag: wallaby-20220524T085900
 ovn_tag: wallaby-20220223T143249
 openvswitch_tag: wallaby-20220223T143249
+prometheus_tag: wallaby-20220525T174700
```

Comment:
Did you rebuild all prometheus images? Should we remove the ones below?

Reply:
The regexp feature is broken, thus all get rebuilt always. Fix in https://github.com/stackhpc/stackhpc-kayobe-config/pull/68
Does ~30 minutes build time of all images match your experience? I saw other jobs that had run for ~4h 30m

Reply:
30 minutes would be pretty quick, even if using the cache.

Reply:
Presumably you at least want to upgrade alertmanager, given you changed the version

Reply:
By default: ```prometheus_alertmanager_tag: "{{ prometheus_tag }}"```
The ones below are pretty custom - I'm not confident they'll work correctly in the newer build, thus leaving them.